.changeset/grumpy-crews-build.md

```diff
@@ -0,0 +1,7 @@
+---
+'example-app': patch
+'@backstage/cli': patch
+'@backstage/config-loader': patch
```

Comment:
This would be a `minor` bump of the `config-loader` only. Stability index for that package is at 1, https://backstage.io/docs/overview/stability-index#config-loaderhttpsgithubcombackstagebackstagetreemasterpackagesconfig-loader, so we should include instructions for how to update existing usage.

Dependent packages will receive an automatic `patch` bump due to the `minor` bump.

Reply:
Should we skip example-app too?

Reply:
Yep. To be clear the breaking change here is the key in the options passed to `schema.process()`, everything else is internal. It's also not something we expect people to be using directly atm since it's wrapped up in other calls in the CLI and `backend-common`. Should still have a short notice about how the parameter should be renamed though.